docs/README.rst

```diff
@@ -417,8 +500,8 @@ Example Configuration Files
 - `example-vs-resource.json <./_static/config_examples/example-vs-resource.json>`_
 - `example-vs-resource-iapp.json <./_static/config_examples/example-vs-resource-iapp.json>`_
 - `example-advanced-vs-resource-iapp.json <./_static/config_examples/example-advanced-vs-resource-iapp.json>`_
-- `single-service-ingress.yaml <./_static/config_examples/single-service-ingress.yaml>`_
-- `single-service-tls-ingress.yaml <./_static/config_examples/single-service-tls-ingress.yaml>`_
+- `single-Service-ingress.yaml <./_static/config_examples/single-Service-ingress.yaml>`_
```

Comment:
Should these instances of 'service' be changed to 'Service'? I'm thinking we may have broken links here.

Reply:
I think Sphinx would automatically convert it to lowercase, but I'll change it anyway.